src/main/resources/static/eiffel-rabbit-mq-post-receive-hook.soy

```diff
@@ -19,12 +19,21 @@
   {/call}
 
   {call aui.form.textareaField}
-    {param id: 'rabbit.exchange' /}
```

Comment:
would be interesting to see if we can do a drop down list here instead of textbox